template/default.tmpl

```diff
@@ -46,9 +46,19 @@ Alerts Resolved:
 {{ define "opsgenie.default.source" }}{{ template "__alertmanagerURL" . }}{{ end }}
 
 
-{{ define "victorops.default.state_message" }}{{ template "__subject" . }} | {{ template "__alertmanagerURL" . }}{{ end }}
-{{ define "victorops.default.monitoring_tool" }}{{ template "__alertmanager" . }}{{ end }}
+{{ define "victorops.default.state_message" }}{{ template "__alertmanager" . }}: {{ template "__alertmanagerURL" . }}
```

Comment:
Understood, I will use the same template used for `opsgenie.default.description`.